docs/Configuration.md

```diff
@@ -1074,15 +1074,15 @@ Setting this value to `fake` allows the use of fake timers for functions such as
 
 ### `transform` [object<string, string>]
 
-Default: `undefined`
+Default: `[{".*": "babel-jest"}]`
 
 A map from regular expressions to paths to transformers. A transformer is a module that provides a synchronous function for transforming source files. For example, if you wanted to be able to use a new language feature in your modules or tests that isn't yet supported by node, you might plug in one of many compilers that compile a future version of JavaScript to a current one. Example: see the [examples/typescript](https://github.com/facebook/jest/blob/master/examples/typescript/package.json#L16) example or the [webpack tutorial](Webpack.md).
 
 Examples of such compilers include [Babel](https://babeljs.io/), [TypeScript](http://www.typescriptlang.org/) and [async-to-gen](http://github.com/leebyron/async-to-gen#jest).
 
 _Note: a transformer is only run once per file unless the file has changed. During development of a transformer it can be useful to run Jest with `--no-cache` to frequently [delete Jest's cache](Troubleshooting.md#caching-issues)._
 
-_Note: if you are using the `babel-jest` transformer and want to use an additional code preprocessor, keep in mind that when "transform" is overwritten in any way the `babel-jest` is not loaded automatically anymore. If you want to use it to compile JavaScript code it has to be explicitly defined. See [babel-jest plugin](https://github.com/facebook/jest/tree/master/packages/babel-jest#setup)_
+_Note: when adding an additional code preprocessors, this will overwrite the default config and `babel-jest` is no longer automatically loaded. If you want to use it to compile JavaScript or Typescript, it has to be explicitly defined by adding `{".(js|jsx)$": "babel-jest"}` or `{".(ts|tsx)$": "babel-jest"}` respectively to the transform array. See [babel-jest plugin](https://github.com/facebook/jest/tree/master/packages/babel-jest#setup)_
```

Comment:
Let's keep the wording consistent, not sure if we call them preprocessors elsewhere
```suggestion
_Note: when adding additional code transformers, this will overwrite the default config and `babel-jest` is no longer automatically loaded. If you want to use it to compile JavaScript or Typescript, it has to be explicitly defined by adding `{".(js|jsx)$": "babel-jest"}` or `{".(ts|tsx)$": "babel-jest"}` respectively to the transform array. See [babel-jest plugin](https://github.com/facebook/jest/tree/master/packages/babel-jest#setup)_
```

Reply:
must of had a mind melt when i wrote preprocessors